contracts/PvpArena.sol

```diff
@@ -871,4 +844,49 @@ contract PvpArena is Initializable, AccessControlUpgradeable {
         //this is not final, but processing the loser will update the ranks leaving this player in the 4th position, which will be quickly replaced by other players
         processLoser(characterID);
     }
+
+    /// @dev distributes the ranking rewards pool to top players
+    // TODO: Re-evaluate if private or requires some permission
+    function _distributeSeasonRewards() private {
+        for (uint8 i = 0; i <= 15; i++) {
+            if (_fightersByTier[i].length() == 0) {
+                continue;
+            }
+            for (uint8 j = 0; j < _prizePercentages.length; j++) {
+                _transferRewards(
+                    _rankingByTier[i][j],
+                    j + 1,
+                    _rankingsPoolByTier[i]
+                );
```

Comment:
What if we only have 2 people in a particular rank? This should only distribute to the players that exist in the rank (which means there's an edge case regarding how the excess prizes are distributed. (maybe just add the excess to the top character's rewards?)